Dockerfile

```diff
@@ -18,15 +18,21 @@ RUN apt install --yes \
         llvm-dev \
         make \
         nodejs \
+        pkg-config \
         protobuf-compiler \
         ragel \
         rustc \
         yarn
 
+ENV GOPATH=/go
+ENV PATH /go/bin:$PATH
+RUN go get github.com/influxdata/pkg-config
```

Comment:
I don't think this is needed. The `Makefile` should take care of using an ephemeral `pkg-config` wrapper. The `pkg-config` package that's installed above is definitely needed. What was the error message when these lines were excluded?

Reply:
Ah, this PR was created before I fixed the hard coded path inside this:

https://github.com/influxdata/influxdb/commit/917fcfc2d7b7a7feea67647ec6f9687a7065f1b7

I can probably remove these lines now, I believe you are correct.